src/components/calculator/ExplanationCard/CalculationBreakdown.tsx

```diff
@@ -0,0 +1,277 @@
+import React from 'react'
```

Comment:
Previously, this file was called `ExplanationCard` and contained the calculation breakdown as well as risk budget settings, wrapped in a `Card`. I moved the `Card` to the parent and cleaned this up to just be the `CalculationBreakdown` since we're moving the risk budget settings.